docs/source/whatsnew_0_8.md

```diff
@@ -0,0 +1,54 @@
+# What's new in 0.8 🎉🎉
+
+- Differentiable neural network topology search
+- Multiple instance learning for digital pathology WSI analysis
+- Self-supervised representation learning modules
+- Major usability improvements in `monai.transforms`
+
+## Differentiable neural network topology search
+MONAI integrates `DiNTS`: [Differentiable Neural Network Topology Search for 3D
+Medical Image Segmentation](https://arxiv.org/abs/2103.15954). The neural
+architecture search module supports flexible multi-path topology search with
+high search efficiency and budgeted memory usage.
+
+It provides a topology guaranteed discretization algorithm and a
+discretization-aware topology loss for the search stage to minimize the
+discretization gap. The module is memory usage aware and is able to search 3D
+networks with different GPU memory requirements. For more details, please checkout the
+[DiNTS tutorial](https://github.com/Project-MONAI/tutorials/tree/master/neural_architecture_search).
+![DiNTS](../images/dints-overview.png)
+
+## Multiple instance learning for digital pathology WSI analysis
+For [classification of digital pathology whole slide images
+(WSI)](https://arxiv.org/abs/2111.01556), MONAI introduces new transforms and
+network modules for multiple instance learning. These include self-attention
+transformer blocks for explicitly accounting of the dependencies between instances
+(image patches) during training. For more details,
+please checkout the [multi-instance tutorial](https://github.com/Project-MONAI/tutorials/tree/master/pathology/multiple_instance_learning)
+
+![multi-instance](../images/mil-patches.jpg)
+
+## Self-supervised representation learning modules
+MONAI starts to explore self-supervised representation learning in this
+milestone release. The Vision Transformer has been extended to learn from self-supervised
+reconstruction tasks with various data augmentation and a regularized
+contrastive loss. The weights of the pre-trained backbone could be used to
+enhance the performance of the novel downstream deep learning tasks.
+
+The [tutorial](https://github.com/Project-MONAI/tutorials/tree/master/self_supervised_pretraining)
+shows how to generate a good set of pre-trained weights using unlabeled data
+with self-supervised tasks, then use the pre-trained weights to perform
+fine-tuning on a fully supervised volumetric segmentation task.
```

Comment:
before the sentence ends, we can add 'using UNETR which is a transformer based model'